pipeline/build-rpm-package.yaml

```diff
@@ -99,6 +99,10 @@ spec:
         OCI registry URL where the pipeline stores trusted artifacts, including
         both resulting artifacts and intermediary by-products.
       default: quay.io/redhat-user-workloads/$(context.taskRun.namespace)/$(params.package-name)
+    - description: Specfile name, default value is null. When specfile is null, package-name will be used as specfile name.
+      name: SPECFILE
```

Comment:
When I see `SPECFILE`, I expect it to be the full specfile name, e.g. `kernel.spec`. But looking at the code below, it seems like the `.spec` suffix is not actually supposed to be passed as part of the name?

Could that be made more obvious? Whether by this variable actually meaning the full specfile name like it seems, and the code treating it as such, or by naming the variable something that makes it more obvious the suffix is not supposed to be there. Unfortunately, naming things is hard and I don't have any obvious suggestions for the rename, as `specfile-prefix` sounds super weird too 🤔